lists/1313161554/list.json

```
@@ -833,6 +833,14 @@
       "decimals": 18,
       "chainId": 1313161554,
       "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0x453E9619b9d5B7062D0d463750fD125a473c65C7/logo.svg"
+    },
+    {
+      "name": "NearX",
+      "address": "0xb39EEB9E168eF6c639f5e282FEf1F6bC4Dcae375",
+      "symbol": "NearX",
+      "decimals": 24,
+      "chainId": 1313161554,
+      "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0xb39eeb9e168ef6c639f5e282fef1f6bc4dcae375/logo.svg"
```

```suggestion
      "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0xb39EEB9E168eF6c639f5e282FEf1F6bC4Dcae375/logo.svg"
```

Replace it by the checksummed address please